paparazzi/paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -738,6 +738,8 @@ class PaparazziPluginTest {
     assertThat(resourceFileContents[1]).isEqualTo("intermediates/merged_res/debug")
     assertThat(resourceFileContents[4]).isEqualTo("intermediates/assets/debug")
     assertThat(resourceFileContents[5]).isEqualTo("app.cash.paparazzi.plugin.test")
+    assertThat(resourceFileContents[6]).isEqualTo("src/main/res/values/strings.xml")
+    assertThat(resourceFileContents[7]).matches("^\\.\\.\\/\\.\\.\\/\\.\\.\\/\\.\\.\\/build\\/tmp\\/test\\/work\\/\\.gradle-test-kit\\/caches\\/transforms-3\\/.+\\/transformed\\/external\\/res\\/values\\/values.xml\$")
   }
```

Comment:
@jrodbx I made the change to use Regex to verify external resource path, the reason is hash of gradle cache is different on local and CI

```
app.cash.paparazzi.gradle.PaparazziPluginTest > verifyResourcesGeneratedForJavaProject FAILED
    expected: …caches/transforms-3/04f769cae83b8eaf5c0db47d9a25d95d/transformed/externa…
    but was : …caches/transforms-3/881c8c88ff6065d763ad7e4873ed32b3/transformed/externa…
        at app//app.cash.paparazzi.gradle.PaparazziPluginTest.verifyResourcesGeneratedForJavaProject(PaparazziPluginTest.kt:742)
```

Reply:
these tests were passing before.  are we sure that the hash wasn't changed when the lib dependency went from src/libs/external.aar to libs/external.aar?

Reply:
I just tested on local and CI and `881c8c88ff6065d763ad7e4873ed32b3` and seems pretty stable...

Reply:
These test failed before I moved the libs. And On my machines it was `04f769cae83b8eaf5c0db47d9a25d95d`, but now it is `881c8c88ff6065d763ad7e4873ed32b3`. Will update the test

Reply:
it went from `04f769cae83b8eaf5c0db47d9a25d95d` to `881c8c88ff6065d763ad7e4873ed32b3` after I fixed spotless. How are they related?

Reply:
it went from `04f769cae83b8eaf5c0db47d9a25d95d` to `881c8c88ff6065d763ad7e4873ed32b3` after I fixed spotless([commit](https://github.com/cashapp/paparazzi/pull/766/commits/5599d2170c3945257a9c07d088bef335b5afb1ad)). How are they related?